src/lint/collect-spelling-diagnostics.ts

```diff
@@ -34,6 +34,23 @@ let matchers = [
     pattern: /[ \t]+\n/gu,
     message: 'Trailing spaces are not allowed',
   },
+  {
+    pattern: /(?<=\n)\n\n+/gu,
+    message: 'No more than one blank line is allowed',
+  },
+  {
+    pattern: /(?<=<emu-clause.*>\n)\n\s*<h1>/giu,
+    message: "There should not be a blank line between a clause's opening tag and its header",
+  },
+  {
+    pattern: /(?<=\n)\n[ \t]*<\/emu-clause>/giu,
```

Comment:
```suggestion
    pattern: /(?<=(^|[^\n])\n)\n\s*<\/emu-clause>/giu,
```

for the same reasons as above.

Reply:
Done, though I used `/(?<=(^|[^\n])\n)\n+[ \t]*<\/emu-clause>/giu` instead of your suggestion because I don't actually want to match vertical tabs or whatever.

Reply:
Didn't add a test for this one because it's impossible to test without also tripping the two-blank-lines rule, and I didn't really want to bother adding support for multiple linting errors.